MapboxDirections/Extensions/String.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+extension String {
+    var isNotEmpty: Bool { return !isEmpty }
```

Comment:
This is probably a bit extreme. For one thing, it’s more verbose than using the `!` operator, which is more versatile than hard-coding negative versions of Boolean properties.